install_files/ansible-base/roles/app/templates/sites-available/journalist.conf

```diff
@@ -2,8 +2,7 @@ ServerName {{ securedrop_app_apache_listening_address }}
 <VirtualHost {{ securedrop_app_apache_listening_address }}:8080>
 
 WSGIDaemonProcess journalist processes=2 threads=30 display-name=%{GROUP} python-path=/var/www/securedrop
-WSGIProcessGroup journalist
-WSGIScriptAlias / /var/www/journalist.wsgi
+WSGIScriptAlias / /var/www/journalist.wsgi process-group=journalist application-group=journalist
```

Comment:
right now this state is enforced via the Ansible playbooks, and so we would need to migrate this on running instances via postinst in the `securedrop-app-code` package

Reply:
I've added this editing to `postinst`, so it should now happen when the package is updated.